lib/routes/routes_page.dart

```diff
@@ -356,14 +58,32 @@ class _RoutesPageBodyState extends State<_RoutesPageBody> {
             padding: const EdgeInsets.only(bottom: 140),
             child: Consumer<FilterChangeNotifier>(
                 builder: (context, filterChangeNotifier, child) {
-              return Column(
-                  children: routes
-                      .where((route) => filterChangeNotifier.filters
-                          .every((filter) => filter.isValid(route)))
-                      .map((ClimbingRoute route) =>
-                          RouteListElement(route: route))
-                      .toList());
-            }))
+              return StreamBuilder<List<RouteData>>(
+                stream: db.routeDao.watchAll,
+                builder: (BuildContext context,
+                    AsyncSnapshot<List<RouteData>> snapshot) {
+                  if (snapshot.hasError) {
+                    return const Text("Error getting routes.");
+                  } else {
+                    switch (snapshot.connectionState) {
+                      case ConnectionState.none:
+                        return const CircularProgressIndicator();
+                      case ConnectionState.waiting:
+                        return const CircularProgressIndicator();
```

Comment:
Have you tested this scenario to confirm if it looks good visually?

Reply:
Yeah, it looks good. I attached a gif so you can let me know what you think.
![studio64_6RfGogRKPC](https://user-images.githubusercontent.com/37987430/188363975-2b2d6210-10bb-4f2d-a4ff-e6d585913809.gif)

Reply:
Could you give this like 20 padding on the top?

Reply:
Also, what do you think about the error message, I really don't anticipate any errors, but I think it looks good enough to move forward. 
![image](https://user-images.githubusercontent.com/37987430/188364794-9ef191ef-2fbb-4449-8857-28525ecf6f8f.png)